tools/translation/translate-challenges.js

```diff
@@ -123,7 +123,7 @@ function processFile(
           let transIndex = 0;
           testsArray.forEach((test, index) => {
             if (test.includes('- text')) {
-              testsArray[index] = '  - text: ' + translation[parseInt(transIndex)];
+              testsArray[parseIn(index)] = '  - text: ' + translation[parseInt(transIndex)];
```

Comment:
Watch out for typos 😉

Reply:
This is outdated comment i have fixed this typo .